### DISCUSSION TITLE
Workspace creation for non-admin users

### DISCUSSION THREAD

**Discussion by heblol** [open-webui/open-webui#6555] [Ideas]
Hi there!

Investigating the code, I saw that the /workspace area is only for admin users, is this true? And if so, might it be an option to make it for non-admin users? (Or maybe introduce another user role that allows for workspace creation)

In the environment variable, maybe more options about the workspace could be created. E.g., allowing a user to a feature individually [Models, Prompts, Documents, Tools, Functions]

Is there a valid reason only to allow admin users to create workspaces?

Thank you!

**Comment by flefevre** [comment]
It will be part of 0.4.0 release that will introduce manager of workspace.
If I do understand

**Reply by heblol** [reply]
Oh oke perfect! Thank you for the quick response. Where did you find that info?

**Reply by flefevre** [reply]
I have made perhaps an error.
I have downloaded last version 0.4.1 and I didn't find a way to create several workspaces and add them a team .

**Reply by flefevre** [reply]
I have written a ticket for enhancement here https://github.com/open-webui/open-webui/releases/tag/v0.4.0
Hope it is clear since I am french and my English not so perfect.

**Reply by heblol** [reply]
Hi flefevre, thank you for the response. I was indeed also checking out the Workspace, however, I don’t think it’s possible yet :(

It would like nice if a user was grated access to a certain group, and then have their own freedom to do whatever they want inside the workspace. Then the models and everything they do is shared with 1. only themselves, 2. the whole group, 3. it’s public and shared with everybody within the website.

**Reply by flefevre** [reply]
Bad copy paste

https://github.com/open-webui/open-webui/discussions/7082#discussion-7519724
